ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/ContentTest.kt

```diff
@@ -139,6 +139,7 @@ class ContentTest : ClientLoader() {
     }
 
     @Test
+    @Ignore
```

Comment:
Maybe just remove them from master?

Reply:
I have no complaints, but I would keep them for the history

Reply:
Git stores all the history

Reply:
Git stores all the history

Reply:
Marking tests ignored gives as more visibility that there is something wrong with them then git history.

Reply:
My impression is that ignoring a test usually means that it may be enabled back. Do we think that the test could be useful in the future?